src/app/zap-templates/common/ClustersHelper.js

```diff
@@ -95,9 +97,10 @@ function loadCommandArguments(command, packageId)
 function loadCommands(packageId)
 {
   const { db, sessionId } = this.global;
-  return queryEndpoint.selectEndPointTypeIds(db, sessionId)
-      .then(endpointTypes => zclQuery.exportClustersAndEndpointDetailsFromEndpointTypes(db, endpointTypes))
-      .then(endpointTypesAndClusters => zclQuery.exportCommandDetailsFromAllEndpointTypesAndClusters(db, endpointTypesAndClusters))
+  return queryEndpointType.selectEndpointTypeIds(db, sessionId)
+      .then(endpointTypes => queryEndpointType.selectClustersAndEndpointDetailsFromEndpointTypes(db, endpointTypes))
+      .then(endpointTypesAndClusters => queryCommand.selectCommandDetailsFromAllEndpointTypesAndClusters(
+                db, endpointTypesAndClusters))
```

Comment:
This is part of the behavior change wrt command ordering.  The old code here grouped commands by name/id.  The new code does not, because `selectCommandDetailsFromAllEndpointTypesAndClusters` now has a third arg controlling that, with "do not group" as default.  Do we want to keep grouping?  @vivien-apple

Reply:
I put the grouping back in for now.  Ideally we'd have an actual ORDER BY in those queries so we get deterministic behavior...